spec/services/hmpps_api/oauth/token_spec.rb

```diff
@@ -5,20 +5,20 @@
     access_token = generate_jwt_token
     token = described_class.new(access_token: access_token, expires_in: 4.hours)
 
-    expect(token).not_to be_expired
+    expect(token).not_to be_needs_refresh
```

Comment:
Would be nice to make this more eloquent and follow the format of the other tests:

```suggestion
    expect(token.needs_refresh?).to be(false)
```